src/components/app.jsx

```diff
@@ -1,4 +1,5 @@
 /* global cozy */
+import 'url-search-params-polyfill'
```

Comment:
Why do we need to import this polyfill? 

Since we target Edge 17+, Safari 11+ and not IE, I think it's pretty safe to remove it